sites/next.skeleton.dev/src/content/docs/tutorials/figma/chapter-2.mdx

```diff
@@ -0,0 +1,150 @@
+import ImgSwitchColorScheme from '@images/tutorials/figma/chapter-2/04_switchColorScheme.webp';
+import ImgSwitchColorSchemeResult from '@images/tutorials/figma/chapter-2/04_switchColorSchemeResult.webp';
+import ImgCopyTheme from '@images/tutorials/figma/chapter-2/05_copyTheme.webp';
+import ImgPasteTheme from '@images/tutorials/figma/chapter-2/06_pasteTheme.webp';
+import ImgPublish from '@images/tutorials/figma/chapter-2/07_publish.webp';
+import ImgPublishUpdates from '@images/tutorials/figma/chapter-2/07_publishUpdates.webp';
+
+---
+
+## Import Custom Theme
+
+<Process>
+    <ProcessStep step="1">
+        ### Open Skeleton v3 Figma Library
+
+        **Important:** The plugin is compatible only with the Skeleton v3 Figma Library.
+        
+        <Image src={ImgOpenSkeleton} class="rounded-container" alt="Create New Project" />
+    </ProcessStep>
+
+    <ProcessStep step="2">
+        ### Open Skeleton Themes plugin
+
+        Click on the `Actions`¹ button in the toolbar, search for `Skeleton Themes`², and click on the `Skeleton Themes`³ plugin when it appears.
+
+        <Image src={ImgOpenThemePlugin} class="rounded-container" alt="Create New Project" />
+    </ProcessStep>
+
+    <ProcessStep step="3">
+        ### Add new theme
+
+        The library includes a default theme, `Cerberus`. To add your theme, click `Add theme`⁴.
+
+        <Image src={ImgAddNewTheme} class="rounded-container" alt="Import Project" />
+    </ProcessStep>
+
+    <ProcessStep step="4">
+        ### Select theme
+
+        Copy your theme code by visiting the [Skeleton Theme Creator](https://themes.skeleton.dev/themes/create). Click `Import`⁵, then select a theme⁶, or compose your own.
+
+        <Image src={ImgSelectTheme} class="rounded-container" alt="Publish" />
+    </ProcessStep>
+
+    <ProcessStep step="5">
+        ### Copy theme
+
+        Open the `Code` tab in the right panel⁷ and copy the theme code⁸.
+
+        <Image src={ImgCopyTheme} class="rounded-container" alt="Create New File" />
+    </ProcessStep>
+
+    <ProcessStep step="6">
+        ### Paste theme
+
+        Return to the `Skeleton Themes` plugin in Figma, paste the theme code⁹, and click `Upload`¹⁰.
+
+        **Note:** Uploading the theme may take 3–5 minutes, depending on your device.
+
+        <Image src={ImgPasteTheme} class="rounded-container" alt="Select Library" />
+    </ProcessStep>
+
+    <ProcessStep step="7">
+        ### Publish updates
+
+        After uploading, publish the library to make the new theme accessible. In the left panel, click the `Book` icon¹¹, then click the `Publish...` button¹². When the modal appears, click `Publish` again¹³.
+
+        **Note:** Publishing may take 5–7 minutes.
+
+        <Image src={ImgPublishUpdates} class="rounded-container" alt="Use Library" />
+        <Image src={ImgPublish} class="rounded-container" alt="Use Library" />
+    </ProcessStep>
+</Process>
+
+---
+
+## Import Custom Theme in Project
+
+### Prerequisites:
```

Comment:
Given we're going to have so many chapters and this is meant to be read in a linear fashion, the requisite list feels redundant. If a step is optional, then indicate it plainly upfront. We do this with titles sometimes too so it's visible in the Table of Contents:

```
1. Do Something
2. Optional: Do Something Else
3. Wrapping up
```

Reply:
Most users do not follow orders (although it would make things so much easier), so we have the prerequisites at the top indicating chapters that are a MUST before you continue to remind all the people who skipped directly to this chapter. You will see in the upcoming tutorials that
1. only a limited amount of chapters are a hard prerequisite
2. we have implemented a fallback system to cover people who jump directly to e.g. Chapter 4 without reading e.g. Chapter 3.

So stay tuned :)